pkg/services/datasources/models.go

```diff
@@ -161,6 +163,7 @@ type AddDataSourceCommand struct {
 	JsonData        *simplejson.Json  `json:"jsonData"`
 	SecureJsonData  map[string]string `json:"secureJsonData"`
 	UID             string            `json:"uid"`
+	IsPrunable      bool
```

Comment:
You also need to add the comment  `swagger:ignore` here and in `UpdateDataSourceCommand`